lib/stateTransition/AbstractStateTransition.js

```diff
@@ -26,19 +27,24 @@ class AbstractStateTransition {
    */
   constructor(rawStateTransition = {}) {
     this.signature = null;
+    this.protocolVersion = getProtocolVersion();
 
```

Comment:
Don’t we need to use `getProtocolVersion` in factories in order to pass the whole raw thing into constructor?